src/features/advanced-content-insights/AdvancedContentInsightsToggle.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import noop from 'lodash/noop';
+import { FormattedMessage } from 'react-intl';
+import messages from './messages';
+import InfoBadge16 from '../../icon/fill/InfoBadge16';
+// @ts-ignore flow import
+import Toggle from '../../components/toggle';
+import Tooltip from '../../components/tooltip';
+import './AdvancedContentInsightsToggle.scss';
+
+interface Props {
+    hasDescriptionInTooltip?: boolean;
```

Comment:
Changed to the suggested name. I prefer the shorter version, but I'm not sure if it's clear that it would only be the description inside that tooltip though. 